### DISCUSSION TITLE
How to handle duplicate column error? [Airflow 1.10.15]

### DISCUSSION THREAD

**Discussion by gavinbains** [apache/airflow#21803] [Q&A]
Hello! Running Airflow 1.10.15

Getting an error after running ariflow. I think it calls an alembic db upgrade under the hood:
```shell
[alembic.runtime.migration] Running upgrade 13eb55f81627 -> 338e90f54d61, More logging into task_instance

# above call throws the following error immediately
psycopg2.errors.DuplicateColumn: column "operator" of relation "task_instance" already exists

# last line of the stack trace: error encapsulated by sqlalchemy as a ProgrammingError
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.DuplicateColumn) column "operator" of relation "task_instance" already exists
```

Trying to track down the culprit, my initial thoughts are to drop the airflow tables. Has anyone seen this before?

**Comment by potiuk** [comment]
I think dropping the tables might be best solution - it is likely that your previous upgrade had been broken in the middle or some manual modificaiton to a table has been applied.

**Comment by julianlgil** [comment]
The same thing happened to me, what I did was change the id that appears in the alembic_version table to the correct one, according to the migration status of the DB. You can see this here: https://airflow.apache.org/docs/apache-airflow/stable/migrations-ref.html
In my case something had happened and it had not been updated and it had stayed at 13eb55f81627 and I changed it to 7939bcff74ba since my previous version was 1.10.9 and I wanted to upgrade to 1.10.15
